include/deal.II/base/mpi.h

```diff
@@ -708,6 +708,11 @@ namespace Utilities
            const T &          object_to_send,
            const unsigned int root_process = 0);
 
+    std::vector<unsigned int>
```

Comment:
can you please add doxygen documentation for the function here?

Reply:
I have added a Doxygen documentation  in commit 434f01437f5f6fa81165fa8550dab5f566c0e9fc.